contracts/src/periphery/cert/interfaces/IEigenDACertVerifier.sol

```diff
@@ -0,0 +1,24 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.9;
+
+import {IEigenDACertVerifierBase} from "src/periphery/cert/interfaces/IEigenDACertVerifierBase.sol";
+import {IVersionedEigenDACertVerifier} from "src/periphery/cert/interfaces/IVersionedEigenDACertVerifier.sol";
+
+import {IEigenDAThresholdRegistry} from "src/core/interfaces/IEigenDAThresholdRegistry.sol";
+import {IEigenDASignatureVerifier} from "src/core/interfaces/IEigenDASignatureVerifier.sol";
+
+import {EigenDATypesV1 as DATypesV1} from "src/core/libraries/v1/EigenDATypesV1.sol";
+
+interface IEigenDACertVerifier is IEigenDACertVerifierBase, IVersionedEigenDACertVerifier {
```

Comment:
I don't think we should delay any further, but this doesn't match what we had discussed with the interface structure 

I think we should do a fast follow to make it match
<img width="665" alt="image" src="https://github.com/user-attachments/assets/a7e89e6b-d179-4185-8fab-0e208fe3b965" />

Reply:
cc @pakim249CAL @samlaf

Reply:
Sorry I didn't loop you into this convo but I brought up a concern with sam that there didn't seem to be a good place to put an IEigenDACertVerifierV3 interface. It seemed better to name it IEigenDACertVerifier instead, and as they become deprecated we can add the version number and dump it into legacy. This is because we don't want to go back to the version numbers in the main folder for periphery, and there should only be one canonical interface that outside integrators should need to refer to. Version numbers would be confusing and would bring questions about which versions they should use.

Reply:
My only issue with "legacy" is it sounds like we no longer support it, whereas our clients and proxy need to be backward compatible with ALL versions >= 3 going forward, because we HAVE to assume some customer is running these versions in prod somewhere, and make sure to not break any of them. Something like "LTS" name makes more sense than "legacy" to me. But we can figure out the proper naming and place to put them later.